test/fuzz/transformation_replace_id_with_synonym_test.cpp

```diff
@@ -2026,11 +2026,10 @@ TEST(TransformationReplaceIdWithSynonymTest,
                    .IsApplicable(context.get(), transformation_context));
 }
 
-// TODO(https://github.com/KhronosGroup/SPIRV-Tools/issues/4345): Improve this
-//  test so that it covers more atomic operations, and enable the test once the
-//  issue is fixed.
 TEST(TransformationReplaceIdWithSynonymTest,
      DISABLED_SignOfAtomicScopeAndMemorySemanticsDoesNotMatter) {
+  //  This test is failed at (IsApplicable) line, because of updates from PR
+  //  #4349.
```

Comment:
This kind of comment doesn't belong in the source code, and we already have TODO comments explaining that the test isn't expected to pass at present, so please remove.

Reply:
Done.